plugins/googleanalytics/src/org/pentaho/di/trans/steps/googleanalytics/GaInputStep.java

```diff
@@ -230,6 +231,7 @@ public boolean init( StepMetaInterface smi, StepDataInterface sdi ) {
     return true;
   }
 
+  @VisibleForTesting
```

Comment:
Remove this line ... see comment above (just mark it protected and add a comment that it is protected vs private for testing purposes...)